README.md

```diff
@@ -586,6 +587,45 @@ crud.len('customers')
 ...
 ```
 
+### Count
+
+`CRUD` supports multi-conditional count, treating a cluster as a single space.
+The same as with `select()` the conditions may include field names or numbers,
+as well as index names. The recommended first condition is a TREE index; this
+helps to reduce the number of tuples to scan. Otherwise a full scan is performed.
+If compared with `len()`, `count()` method scans the entire space to count the
+tuples according user conditions. This method does yield that's why result may
+be approximate. Number of tuples before next `yield()` is under control with
+option `count_to_yield`.
+
+```lua
+local result, err = crud.count(space_name, conditions, opts)
+```
+
+where:
+
+* `space_name` (`string`) - name of the space
+* `conditions` (`?table`) - array of [conditions](#select-conditions)
+* `opts`:
+  * `count_to_yield` (`?number`) - number of tuples processed to yield after,
+    `count_to_yield` should be > 0, default value is 100
+  * `timeout` (`?number`) - `vshard.call` timeout (in seconds), default value is 2
+  * `bucket_id` (`?number|cdata`) - bucket ID
+  * `mode` (`?string`, `read` or `write`) - if `write` is specified then `count` is
+    performed on master, default value is `read`
+  * `prefer_replica` (`?boolean`) - if `true` then the preferred target is one of
+    the replicas, default value is `false`
+  * `balance` (`?boolean`) - use replica according to
+    [vshard load balancing policy](https://www.tarantool.io/en/doc/latest/reference/reference_rock/vshard/vshard_api/#router-api-call),
+    default value is `false`
```

Comment:
The following `crud.select()` options look meaningful for `crud.count()` as well:

* `after`: because it is quite meaningful question: how much tuples after given one remains?
* `first`: because we can just make sure that, say, at least 1000 more tuples after current one are present. We are not interested, whether it is 1000 or 5000.
* `force_map_call`: if a user suspect a problem around sharding and want to re-verify with the more durable way.

What is not applicable here:

* `batch_size`: because the result is always tiny.
* `fields`: because we don't return any fields.

I propose to file issues regarding possibly useful options and go ahead.

Reply:
Now there is an issue #264 for that.

Reply:
Nice!